assignment.md

```diff
@@ -14,4 +14,11 @@ Steps:
 
 The output of this assignment should be the command needed to pull and run the necessary image.
 
-Output: `<output goes here>`
\ No newline at end of file
+Output: `<output goes here>`
+`<docker pull hello-world>`
+`<docker run hello-world>`
+`<docker rmi hello-world>`
```

Comment:
I would probably run `docker ps -a` to see all the containers built. Then, run `docker rm <container ID/name>`. Finally, run `docker rmi <image ID>.

Pruning removes all stale images.  If you do this, you'll remove _all_ of your containers.  Sometimes, we won't want to do that.